packages/validator/src/schemas/openapi/v2.0.0.json

```
@@ -948,13 +948,13 @@
           "$ref": "http://json-schema.org/draft-07/schema#/properties/maximum"
         },
         "exclusiveMaximum": {
-          "$ref": "http://json-schema.org/draft-07/schema#/properties/exclusiveMaximum"
+          "type": "boolean"
```

swagger 2 is based on draft-4, that is why the definition is different (http://json-schema.org/draft-04/schema#/properties/exclusiveMaximum). So, it might affect other constraints too (https://github.com/NeuraLegion/json-schema-migrate/blob/master/src/index.ts)

```suggestion
          "type": "boolean",
          "default": false
```

---

i see. and yes, "might affect others", but do not see other similar properties related breaking changes in [changelog](https://json-schema.org/draft-06/json-schema-release-notes.html); thus imo backward migration should not be done